microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md

```diff
@@ -0,0 +1,180 @@
+---
+description: Focus Mode and a vertical Activity Bar.  Auto-minimizing of the Console.  Display a webpage in Visual Studio Code, emulate devices, and see issues while editing.  Sources tool notifies you when sourcemaps can't be loaded.  If Sources is open, it's used rather than Visual Studio Code.
+title: What's new in DevTools (Microsoft Edge 96)
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 11/16/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+# What's New in DevTools (Microsoft Edge 96)
+
+The following sections list the announcements from the Microsoft Edge DevTools team.  To try the latest features of DevTools and the Microsoft Edge DevTools extension for Visual Studio Code, read these announcements.  To stay up to date with the latest and greatest features in your developer tools, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) and [follow the Microsoft Edge DevTools team on Twitter](https://twitter.com/EdgeDevTools).
+
+If you are on Windows, Linux, or macOS, consider using the Canary preview channel of Microsoft Edge as your default development browser.  The preview channels give you access to the latest features of Microsoft Edge DevTools.
+
+
+<!-- ====================================================================== -->
+## New DevTools UI available (in preview)
+
+<!-- 1 -->
+
+<!-- Title: New DevTools UI available (in preview) -->
+<!-- Subtitle: A more minimal, modern UI is coming to Microsoft Edge DevTools. Enable the "Focus Mode" experiment to preview new UI features such a more compact toolbar that keeps DevTools uncluttered and better adapts to small window sizes. -->
+
+The Microsoft Edge DevTools team is experimenting with a new DevTools UI: **Focus Mode**, which provides a vertical Activity Bar for all of your tools.  This Activity Bar gives a quick view, to have two tools open simultaneously, and provides several customization and organization options to personalize your tools.
```

Comment:
It is vertical by default, but you can also turn it horizontal, so I guess removing it might help.

Reply:
Left requested revisions below.